contrib/node/src/python/pants/contrib/node/tasks/node_bundle.py

```diff
@@ -0,0 +1,61 @@
+# coding=utf-8
+# Copyright 2015 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import (absolute_import, division, generators, nested_scopes, print_function,
+                        unicode_literals, with_statement)
+
+import os
+
+from pants.base.build_environment import get_buildroot
+from pants.fs import archive
+
+from pants.contrib.node.tasks.node_paths import NodePaths
+from pants.contrib.node.tasks.node_task import NodeTask
+
+
+class NodeBundle(NodeTask):
+  """Create an archive bundle of NodeModule targets."""
+
+  @classmethod
+  def register_options(cls, register):
+    super(NodeBundle, cls).register_options(register)
+    # Some node modules depend on symlink to function.  Thus we can only support archival type
+    # that can preserve symlinks.
+    register('--archive', choices=list(archive.TYPE_NAMES_PRESERVE_SYMLINKS),
+             default='tgz',
+             fingerprint=True,
+             help='Create an archive of this type.')
+    register('--archive-prefix', type=bool, default=False,
+             fingerprint=True,
+             help='If --archive is specified, use the target basename as the path prefix.')
+
+  @classmethod
+  def product_types(cls):
+    return ['node_bundles']
```

Comment:
You're not currently populating this product, but it would be super cool if you did. That would allow other tasks to depend on the bundles you've created to, say, upload them places.

Also, take a look at https://github.com/pantsbuild/pants/blob/master/src/python/pants/backend/jvm/tasks/bundle_create.py#L66 ... creating the `deployable_archives`  product would be neat as well (less specific: accounts for all bundles that pants is capable of producing: jvm, python, etc).